library/bignum_core.h

```diff
@@ -0,0 +1,158 @@
+/**
+ *  Core bignum functions
+ *
+ *  This interface only should be used by the legacy bignum module (bignum.h)
```

Comment:
It's more usual to have `only` and `should` the other way round
```suggestion
 *  This interface should only be used by the legacy bignum module (bignum.h)
```